### DISCUSSION TITLE
如何新建脚本啊？

### DISCUSSION THREAD

**Discussion by aziper3** [scriptscat/scriptcat#180] [Q&A] [answered]
如何新建脚本并且运行啊

**Comment by CodFrm** [comment, marked as answer]
很容易呀

<img width="331" alt="image" src="https://user-images.githubusercontent.com/22783163/235937807-83bd9fbf-7516-4fd8-86f3-803382773c46.png">

<img width="281" alt="image" src="https://user-images.githubusercontent.com/22783163/235937872-747cde6e-c714-45dc-b860-0453aea09a94.png">
